test/pylib/manager_client.py

```diff
@@ -100,6 +100,14 @@ async def _get_text(self, resource: str) -> str:
         resp = await self._get(resource)
         return await resp.text()
 
+    async def _get_with_driver_refresh(self, resource: str) -> None:
```

Comment:
The name should be `_get_text_with_driver_refresh`
but this approach is quite limiting anyway.
I would recommend writing a higher order function `with_driver_refresh` which takes the actual request function as a parameter and adds`close`+`connect` around it. Then you would do something like:
`self.with_driver_refresh(_get_text, "/the/path")`

Reply:
Done in [latest push](https://github.com/scylladb/scylladb/commit/8e0fbc433bdac65cf53c66170106baf2b13434e7)